yuniql-cli/CommandLineService.cs

```diff
@@ -40,7 +40,7 @@ private Configuration SetupRunConfiguration(BaseRunPlatformOption opts, bool isV
             var tokens = opts.Tokens.Select(t => new KeyValuePair<string, string>(t.Split("=")[0], t.Split("=")[1])).ToList();
 
             configuration.Workspace = opts.Workspace;
-            configuration.IsDebug = opts.IsDebug;
+            configuration.IsDebug = opts.IsDebug?? false;

```

Comment:
Same comment as above. Because of the change, you have to follow along this with more changes. I thinks its not necessary but please justify this change.

Reply:
Answer is above !